n2kMappings.js

```diff
@@ -12,6 +12,39 @@ function getMmsiContext(n2k) {
 }
 
 exports.mappings = {
+  //Battery Voltage
+  '127508': [{
+    source: 'Voltage',
+    node: 'electrical.batteries.0.capacity.stateOfCharge',
+    filter: function(n2k) {
+      return n2k.fields['Battery Instance'] === 0;
+    }
+  }, 
+{
+    source: 'Voltage',
+    node: 'electrical.batteries.1.capacity.stateOfCharge',
```

Comment:
Instead of having two "rules" there could be just one that uses the instance field as the id. Sort of like calculationType is handled below. This would support more than 2 batteries.